oncopipe/oncopipe/__init__.py

```diff
@@ -1242,11 +1243,18 @@ def generate_runs(
                     (samples.sample_id == normal_id) & (samples.seq_type == seq_type)
                 ]
                 num_matches = len(normal_row)
-                assert num_matches == 1, (
+
+                if (
+                    num_matches == 0
+                ):
+                    print(
                     f"There are {num_matches} {seq_type} samples matching "
-                    f"the normal ID {normal_id} (instead of just one)."
-                )
-                unmatched_normals[key] = Sample(*normal_row.squeeze())
+                    f"the normal ID {normal_id}. Make sure the default unmatched normal for {key} specified in "
+                    f"config[‘unmatched_normal_ids’] is not excluded from the samples table"
+                    )
+                    quit()
+                elif num_matches >= 1:
```

Comment:
Are you 100% sure there's no way this will break in changing it from the current `==1` condition? I would be more comfortable if this was `elif num_matches ==  1` and there was an `else` statement for if num_matches is not 0 or 1.

Reply:
I'll add that `>1` scenario here, sure. Do we want it to exit with an error or give a warning? I think the error is preferable, since this would mean there are several samples with the same sample_id and seq_type--genome build mapping, and also error will be consistent with existing functionality.

Reply:
@lkhilton suggestion has my vote as well, just to be safe. An error message seems to be the more appropriate alternative.

Reply:
I agree an error is better! Thanks.

Reply:
Better to never assume we can't get a certain condition and always handle it. Python exceptions when not caught are often useless.

Reply:
Thank you all for the suggestions! The check for `>1` is now added and will result in the error message like this:
```
There are 2 capture samples matching the normal ID TCRBOA7-N-WEX. This means there are 2 normal samples for capture--grch37 in the samples table and it is not desired. Please ensure all sample_id, seq_type, and genome_build combinations are unique.
```
This update was tested to behave normally in GAMBL 🙌